website/docs/layouts/app-frame/partials/code/component-api.md

```diff
@@ -77,6 +77,9 @@ To be used as container for the application's main page content.
   <C.Property @name="yield">
     Elements passed as children are yielded as inner content of a `<main>` HTML element.
   </C.Property>
+  <C.Property @name="id" @default='"main"'>
```

Comment:
Should we add an example of a custom a11yRefocusLink to the appFrame how to use section? Could be helpful to have 1 place where all these parts come together on our docs instead of just updating the API for each component.

Reply:
I added an explanatory section to the AppFrame "How to use" docs.

Reply:
Btw, I can also refine the docs further as part of the phase 2 Enterprise navigation work.